Handle West Virginia brown eye color.

General-Fault opened the pull request:
Fixes 21

c0shea commented on IdParser/Parsers/Id/EyeColorParser.cs:
```diff
@@ -38,6 +38,11 @@ public override void ParseAndSet(string input)
             {
                 IdCard.EyeColor = EyeColor.Brown;
             }
+            // West Verginia doesn't follow the abbreviation scheme for brown
```
Can you fix the Virginia typo? Also, it's probably a good idea to add this same BN logic to the `HairColorParser`.

General-Fault commented:
Fixed the embarrassing typo and added the alternate abbreviation for brown.

Would it make sense to add an "alternate" IEnumerable<string> to the Abbreviation attribute to handle this more centrally? Be happy to do so if you think this is worthwhile. 

```
     /// <summary>
    /// Specifies an abbreviated display value for the target.
    /// </summary>
    [AttributeUsage(AttributeTargets.All)]
    public class AbbreviationAttribute : Attribute
    {
        public string Abbreviation { get; }
		public IEnumerable<string> AlternateAbbreviations { get; }

	    public AbbreviationAttribute(string abbreviation,  params string[] alternateAbbreviations)
	    {
		    Abbreviation = abbreviation;
		    AlternateAbbreviations = alternateAbbreviations ?? Enumerable.Empty<string>();
	    }

	    public AbbreviationAttribute(string abbreviation) : this (abbreviation, null)
        {}
    }
```

Then the "brown" enum could be defined as
```
        [Abbreviation("BRO", "BRN", "BR", "BN")]
        Brown,

```

The extension methods would need to be updated to something like this:

```
        /// <summary>
        /// Gets the value of the <see cref="AbbreviationAttribute"/> on the <see cref="Enum"/>.
        /// </summary>
        public static string GetAbbreviation(this Enum value)
        {
            return value.GetAttributeValueOrDefault<AbbreviationAttribute, string>(a => a.Abbreviation, value.ToString());
        }

	    public static bool IsAbbreviationForValue(this Enum value, string abbreviation)
	    {
		    return abbreviation.EqualsIgnoreCase(value.GetAttributeValueOrDefault<AbbreviationAttribute, string>(a => a.Abbreviation, value.ToString()))
		           || (value.GetAttributeValueOrDefault<AbbreviationAttribute, IEnumerable<string>>(a => a.AlternateAbbreviations)?.Any(abbreviation.EqualsIgnoreCase) ?? false);

	    }

```

General-Fault commented:
Ugh! Made a mess of my branches.  Created new push request #23.
